### ISSUE TITLE
Multiple specifications for "no precipitation" (Bugzilla Bug 487)

### ISSUE THREAD

**Issue by datexii** [DATEX-II-EU/DatexII#487]
This issue was created automatically with bugzilla2github.py
# Bugzilla Bug 487
Date: 2024-06-03T11:36:08+02:00
From: @JoergFst
To:   @JoergFst

Last updated: 2024-06-03T11:36:08+02:00

**Comment by JoergFst** [comment]
Agreed to remove boolean attribute in class PrecipitationInformation, which is a Part 5 issue (see #563).
Closed for Part 7 (no further action in Part 7).
